doc/changelog.txt

```diff
@@ -91,6 +91,7 @@ Sim:
    will still always target enemies and never allies, so avoid using it outside of manually-triggered contexts
  - nanoturret (immobile builder) build-range now only needs to reach the edge of the buildee's radius
    instead of its center. Mobile builders already worked this way.
+ - Remove redundant optional `showDeathSequence` parameter from `KillUnit()` definitions.
```

Comment:
I wouldn't put this in the changelog since it's just internal engine cleanup. Game devs don't interact with this at all

Reply:
Noted, removing changelog entry.